tfx/components/example_gen/write_split.py

```diff
@@ -48,22 +52,36 @@ def process(self, e: Union[tf.train.Example, tf.train.SequenceExample,
 
 @beam.ptransform_fn
 @beam.typehints.with_input_types(
-    Union[tf.train.Example, tf.train.SequenceExample, bytes])
+    Union[tf.train.Example, tf.train.SequenceExample, bytes, pa.Table])
 @beam.typehints.with_output_types(beam.pvalue.PDone)
 def WriteSplit(
     example_split: beam.pvalue.PCollection,
     output_split_path: str,
     output_format: str,
```

Comment:
can we use the output_format? which should contains parquet files too.
or we can add output_payload_format here to align with output_format instead of exec_properties

(somehow I though exec_properties is already in param list before)

Reply:
we also need the exec_properties to get the schema out of it for writing. I can also create a separate function for writing parquet splits, they share very little logic